content/docs/custom-commands/commands.md

```diff
@@ -242,6 +252,61 @@ The modal trigger is used to trigger custom commands via submitting a modal.
 
 The trigger is matched using [RegEx](/docs/reference/regex).
 
+##### Crontab
+
+This trigger will run the command at scheduled intervals using [Cron Job Scheduling](https://en.wikipedia.org/wiki/Cron). This allows for advanced specificity in scheduling execution, such as scheduling execution at 23:45 every Saturday.
```

Comment:
I think we should avoid using the term "scheduled intervals" in this section to avoid confusion with interval triggers. Drawing an explicit contrast to interval triggers would also be nice. I propose

"This trigger runs the command periodically at times specified using [cron syntax]. In contrast to interval triggers, which run a command with a fixed delay but unknown time, cron triggers can be made to execute at fixed times of day. For instance, a cron trigger with the expression `45 23 * * 7` runs every Saturday at 23:45 UTC."

I'm not particularly fond of the phrase "execute at fixed times of day", given that cron triggers really have more flexible applications, but I have nothing better at the moment.